addons/stock_account/models/stock_move.py

```diff
@@ -465,6 +465,7 @@ def _should_create_account_move(self):
         :return: True if an account move should be created, False otherwise.
         """
         self.ensure_one()
+        # TODO: check
```

Comment:
I added this comment while rebasing the branch.  As it seems to work fine, we can remove it.